extendeddatacrossword.go

```diff
@@ -249,7 +247,6 @@ func (eds *ExtendedDataSquare) solveCrosswordCol(
 // 2. Whether the original shares could be decoded from the shares parameter.
 // 3. [Optional] an error.
 func (eds *ExtendedDataSquare) rebuildShares(
```

Comment:
do we need this function at all if its just decoding?

Reply:
good question, I can explore deleting it entirely.

Reply:
TBH I'm only creating a new issue rather than tackling in this PR b/c I don't want to dismiss existing approvals 😞 https://github.com/celestiaorg/rsmt2d/issues/199